docs/track.rst

```diff
@@ -3132,6 +3132,40 @@ Meta-data
 
 The operation returns no meta-data.
 
+esql
+~~~~~~~~~~~~~
+
+With the operation type ``esql`` you can execute `ES|QL query <https://www.elastic.co/guide/en/elasticsearch/reference/master/esql.html>`_.
+
+Properties
+""""""""""
+
+* ``query`` (mandatory): An ES|QL query starts with a source command followed processing commands.
+* ``filter`` (optional): A query filter defined in `Elasticsearch query DSL <https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html>`_.
+* ``body`` (optional): The query body.
```

Comment:
The description here should indicate that this is used for anything additions to put into the body, since the query will already be placed there. For example, we use this for `pragma`.